.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "googleapis"]
```

Comment:
Does the .gitmodules file allow comments in it? If so, clarify that this is just used for tests.